cms/static/js/views/list_textbooks.js

```diff
@@ -32,7 +32,15 @@ define(["js/views/baseview", "jquery", "js/views/edit_textbook", "js/views/show_
         },
         addOne: function(e) {
             if(e && e.preventDefault) { e.preventDefault(); }
-            this.collection.add([{editing: true}]);
+            this.collection.add([{editing: true}]); // (render() call triggered here)
+            // find the outer 'section' tag for the newly added textbook
+            $sectionEl = this.$el.find('section:last');
+            // scroll to put this at top of viewport
+            ViewUtils.setScrollOffset($sectionEl, 0);            
+            // find the first input element in this section
+            $inputEl = $sectionEl.find('input:first');
+            // activate the text box (so user can go ahead and start typing straight away)
+            $inputEl.focus().select();
```

Comment:
Yes focus() would be sufficient currently, since the text box is created empty. I included the select() to be consistent with other code, and because this seems like the best behaviour in case some default text gets added later on.